staging/src/k8s.io/kubectl/pkg/kuberc/kuberc.go

```diff
@@ -96,11 +97,16 @@ func (p *Preferences) Apply(rootCmd *cobra.Command, args []string, errOut io.Wri
 		return args, nil
 	}
 
+	verbosity, err := getLogVerbosity(args)
```

Comment:
I feel like this belongs in [NewKubectlCommand](https://github.com/kubernetes/kubernetes/blob/1b517e55013479f674f3fac7196f3af307d6e23f/staging/src/k8s.io/kubectl/pkg/cmd/cmd.go#L311) since this way we'll be able to use it in the entire kubectl initialization rather than just only in here.